docs-site/docs/autograd.md

```diff
@@ -68,6 +68,13 @@ For those operations that require parameters, we package them into a new class,
 `Layer` manages (stores) the parameters and calls the corresponding `Operation`s
 to implement the transformation.
 
+### Module
```

Comment:
Module vs Model?
Does pytorch use Module?

Reply:
Yes, PyTorch uses Module rather than Model.

Reply:
Like this. [Image classification (MNIST) using Convnets](https://github.com/pytorch/examples/blob/master/mnist/main.py)